synapse/models/person.py

```diff
@@ -178,6 +212,20 @@ def getBaseModels():
                     ('seen:min', {'ptype': 'time:min'}),
                     ('seen:max', {'ptype': 'time:max'}),
                 )),
+
+                ('ps:has', {}, [
+                    ('person', {'ptype': 'ps:person', 'ro': 1, 'req': 1,
+                        'doc': 'The action that references the given node.'}),
+                    ('xref', {'ptype': 'propvalu', 'ro': 1, 'req': 1,
+                        'doc': 'The prop=valu that is referenced as part of the action.'}),
```

Comment:
this is ownership, not an action

Reply:
Yep I need to clean up all the docstrings

Reply:
fixed